src/main/java/com/resource/TwitterWorks.java

```diff
@@ -0,0 +1,60 @@
+package com.resource;
+
+import org.eclipse.jetty.util.StringUtil;
+import twitter4j.TwitterException;
+import javax.ws.rs.*;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+@Produces(MediaType.APPLICATION_JSON)
+@Path("/api/1.0/twitter")
+public class TwitterWorks {
+    @POST
+    @Path("/postTweet")
+    @Consumes(MediaType.APPLICATION_JSON)
+    public Response sendTweet(TweetPostRequest tweetPostRequest){
+        String tweet=tweetPostRequest.getMessage();
+        System.out.println(tweet);
+        if(StringUtil.isEmpty(tweet)) {
+            return Response.status(400,"Please Enter a valid tweet").build();
+        }
+        else {
+            try {
+                SendTweet.sendTweets(tweet);
+            }
+            catch (TwitterException e)
+            {
+                return Response.status(500,"Request Was Not Completed").build();
+            }
+            return Response.status(200,"Tweeted Successfully").build();
+        }
+    }
+    @GET
+    @Path("/getTimeline")
+    public String[] timeLine()
+    {
+           int size=RetrieveTweets.latestTweet().length;
+            String ar[]=new String[size];
+            int i=0;
+            for(String s:RetrieveTweets.latestTweet())
+            {
+                ar[i]=s;
+                i+=1;
+            }
+            return ar;
+    }
+}
+class TweetPostRequest
```

Comment:
Add a new file for this, and shift this class to that java file

Reply:
yes i created a new java file TweetPostRequest.java and class TweetPostRequest added in it